play/apps/tournament/models.py

```diff
@@ -59,6 +47,22 @@ def snakes(self):
             snakes.append(st.snake)
         return snakes
 
+    def get_csv(self):
```

Comment:
maybe rename this to export, and not use the implementation in the name, because if it changes at some point then we don't need to update anywhere this is called